pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -276,3 +297,13 @@ func hasOffering(instanceType *cloudprovider.InstanceType, requirements scheduli
 	}
 	return false
 }
+
+func FindRequirementKeyInCompatibleWithMinValues(cumulativeMinRequirementsFromInstanceTypes map[string]sets.Set[string], requirements scheduling.Requirements) string {
```

Comment:
```suggestion
func FindRequirementKeyIncompatibleWithMinValues(cumulativeMinRequirementsFromInstanceTypes map[string]sets.Set[string], requirements scheduling.Requirements) string {
```

nit: Same comment here on incompatible

Reply:
Addressed.